python/test/gluon/test_frontend.py

```diff
@@ -2229,3 +2229,71 @@ def test_layout_zeros():
     # CHECK: #blocked = #ttg.blocked
     # CHECK: arith.constant dense<0.000000e+00> : tensor<128xf32, #blocked>
     ttgl.zeros([128], ttgl.float32, layout=ttgl.BlockedLayout([1], [32], [4], [0]))
+
+
+@pytest.mark.parametrize("target", [HIP_TARGET_CDNA3, HIP_TARGET_CDNA4])
```

Comment:
Can we also add a runtime test in `test_core.py`? Existing Triton level tests can not always guarantee going down buffer atomic path.